src/simple-length.js

```diff
@@ -26,6 +26,8 @@
           this.value = nValue;
         }
       }
+    } else {
+      throw new TypeError('\'' + type + '\' is not a valid type for a SimpleLength.');
     }
     if (this.value == undefined) {
       throw new TypeError('Value of SimpleLength must be a number or a numeric string.');
```

Comment:
While you're here, could you move this check up to the top? It's kind of weird to have it down here, especially once you have the check you've just added.